main.py

```diff
@@ -185,3 +188,31 @@ def get_cookie(dpid):
         """Get 8-byte integer cookie."""
         int_dpid = int(dpid.replace(":", ""), 16)
         return (0x00FFFFFFFFFFFFFF & int_dpid) | (settings.COOKIE_PREFIX << 56)
+
+    def set_flow_table_group_owner(self, flow: dict, group: str) -> dict:
+        """Set owner, table_group and table_id
+        coloring is only allowing 'base' for now"""
+        flow["owner"] = "coloring"
+        flow["table_group"] = group
+
+        # If "base" in table_group, every flow is in one table
+        if "base" in self.table_group:
+            flow["table_id"] = self.table_group["base"]
+            return flow
+        flow["table_id"] = self.table_group.get(group, 0)
+        return flow
+
+    # pylint: disable=attribute-defined-outside-init
+    @alisten_to("kytos/of_multi_table.enable_table")
```

Comment:
This event hasn't been documented on README.md yet, can you update it?

Reply:
Updated